### PR TITLE
fortran used by bench and test clients

### PR DESCRIPTION
**Pull request by TorreZuk**
* clients link and so use Fortran
* gfortran is one pulled as dependency in install.sh so assume it should be default

### CODE REVIEW

**Comment by eidenyoshida** [comment]
the RHEL9 installation failure can be ignored. We have fixed this in math-ci

**Comment by TorreZuk** [comment]
Well @eidenyoshida can you approve, need 2 approvals. But the status for tests seems delayed, runs are done but it still shows "beginning installation stage...."

**Review by eidenyoshida — DISMISSED** [review]

File: clients/benchmarks/CMakeLists.txt
Comment (davidd-amd) [inline clients/benchmarks/CMakeLists.txt:89]: Will the following not work?

```suggestion
   find_package(LAPACK Required)
   target_link_libraries(hipblaslt-bench PRIVATE LAPACK::LAPACK)
```

Comment (TorreZuk) [reply clients/benchmarks/CMakeLists.txt:89]: The cmake config varies so much it wasn't reliable in the past, nor do we want to add an entire file of config rules just to find it.  This was meant as a quick fix but is dragging on due to other CI failures.   Could try to move to a find_package fallback (as AOCL will become default) but I think we will ask @NaveenElumalaiAMD to remove the lapack dependency entirely like done in rocblas.

Comment (TorreZuk) [reply clients/benchmarks/CMakeLists.txt:89]: I think Eiden you should become a code owner, and so only 1 team member review required?

**Review by KKyang — APPROVED** [review]

**Comment by TorreZuk** [comment]
@KKyang can you reapprove as Eiden accidentally requested you review

**Comment by eidenyoshida** [comment]
Merging as it has passed the sles compilation stage that this PR is targeting

**Pull request merged**
